infoschema/tables_test.go

```diff
@@ -1362,6 +1362,31 @@ func (s *testTableSuite) TestStmtSummarySensitiveQuery(c *C) {
 		))
 }
 
+func (s *testTableSuite) TestSimpleStmtSummaryEvictedCount(c *C) {
+	now := time.Now().Unix()
+	interval := int64(1)
+	tk := s.newTestKitWithPlanCache(c)
+	tk.MustExec(fmt.Sprintf("set global tidb_stmt_summary_refresh_interval = %v", interval))
+	tk.MustExec("set global tidb_enable_stmt_summary = 0")
+	tk.MustExec("set global tidb_enable_stmt_summary = 1")
+	tk.MustExec("set global tidb_stmt_summary_max_stmt_count = 1")
+	// first sql
+	tk.MustQuery("show databases;")
+	// query `evicted table` is also a SQL, passing it leads to the eviction of the previous one.
```

Comment:
The comment is wrong. 
The first evicted SQL is `set global tidb_stmt_summary_max_stmt_count = 1`.
The second evicted SQL is `show databases;`.